src_cam/mo_neu_wetdep.F90

```diff
@@ -282,8 +310,11 @@ subroutine neu_wetdep_tend(lchnk,ncol,mmr,pmid,pdel,zint,tfld,delt, &
   real(r8) :: lats(pcols)
 
   ! OSLO_AERO begin
-  real(r8) :: wrk_wd(pcols)
-  logical history_aerosol
+  real(r8)    :: wrk_wd(pcols)
+  logical     :: history_aerosol
+  integer     :: l_aero
+  type(physics_buffer_desc),  pointer :: pbuf(:)
```

Comment:
This should really be up with the rest of the dummy arguments, not under local variables.
While you are at it, you can remove line 267 (`implicit none`), it is redundant.